### PR TITLE
improve readme to include more warnings, improved workflow, and add FAQs

### PR DESCRIPTION
**Pull request by mckennapsean**

### CODE REVIEW

**Comment by mckennapsean** [comment]
I added in a few more warnings / heads-up, based on issues people have filed, along with a FAQ's section. We could add one more about the CORS issues, though I am not sure I am 100% on the complete answer to that yet.

I also changed the main URL and protocol to copy to hopefully reduce the number of people "requesting access to the spreadsheet", since this buzzes my phone and sends an email every time someone does this (with no way to disable it, silly Google).

**Comment by nelsonic** [comment]
@mckennapsean please _assign_ the PR when you feel it's ready for merging. 👍 
(_looking good so far BTW!_) 🎉

File: README.md [hunk 1]
```diff
@@ -323,6 +342,37 @@ if (validateHuman(data.honeypot)) {  //if form is filled, form will not be submi
 ```
 
 
+## Frequently Asked Questions (FAQ's)
+
+1. _How can I get help using this tutorial?_
+
+- This tutorial is meant to help people learn how to use web technologies and make simple, customizable forms with the Google Script API. If you have been unable to get a working example of your code, please check each of the tutorial steps first. You may have missed or glossed over a detail, and this can save both you, and us, time, when trying to help you. Furthermore, if you do this and are still unable to fix your problem, feel free to submit a GitHub issue, but try to include an example of the problem. Highlight what steps you have gone through, what exactly isn't working and why it isn't working. What do you expect to happen? Having an example of the problem, say a GitHub pages or a CodePen, goes a long way since it lowers the barrier for someone to take your code and see what the issue is. If you want to get a faster response, I would highly encourage putting together an example.
```
Comment (iteles) [inline README.md]: Minor minor things here:
+ This is a massive paragraph which is harder to edit/comment on - you might want to consider semantic line breaks (or breaking at the 80 character line mark) which won't affect the visual side of this but will make the markdown easier to follow
+ We've found it's often fruitful to link 'submit a github issue' to https://github.com/dwyl/html-form-send-email-via-google-script-without-server/issues/new for ease of access, though I'm not 'ease of access' is what you're hoping for here! 😆

File: README.md [hunk 2]
```diff
@@ -323,6 +342,37 @@ if (validateHuman(data.honeypot)) {  //if form is filled, form will not be submi
...
+
+2. _Can I get edit access to the example spreadsheet?_
+
+- No. This is being used to show a working example for anyone to copy, and an editable version could be broken accidentally, or maliciously, by any user.
+
+3. _Why is the webpage forwarding to a bunch of text when I hit submit?_
```
Comment (iteles) [inline README.md]: 👍 This is a big one for people who are coming from a world of just HTML and CSS!

File: README.md [hunk 3]
```diff
@@ -323,6 +342,37 @@ if (validateHuman(data.honeypot)) {  //if form is filled, form will not be submi
...
+
+5. _The webpage is saying my data was submitted, but why isn't my data being saved or sent to me?_
+
+- When you copied the spreadsheet and published the Google Script, did you set the permissions to "Anyone, even Anonymous"? This is required for the form to work, since anyone on the internet can hit send to give you their data. Be sure that you have deployed the proper version of the script and used "Manage versions..." when making changes.
+
+6. _How can I upload files?_
+
+- Unfortunately, this feature is not currently supported at this time. It may be possible to tap into the Google Drive API via the Google Script, to save files that were uploaded. We would encourage anyone who has a working example to submit a PR or post an issue with how they solved this.
```
Comment (iteles) [inline README.md]: 👍 for the encouragement

File: README.md [hunk 4]
```diff
@@ -323,6 +342,37 @@ if (validateHuman(data.honeypot)) {  //if form is filled, form will not be submi
...
+
+7. _Is this secure? Can I use it for sensitive data?_
```
Comment (iteles) [inline README.md]: Excellent disclaimer!

File: README.md [hunk 5]
```diff
@@ -272,13 +281,23 @@ your form because it automatically re-loads the page when you make changes in yo
 
 If you want us to take this tutorial further, [***please let us know***!](https://github.com/nelsonic/html-form-send-email-via-google-script-without-server/issues)
 
+For your convenience, we have hosted a working demo of the field on GitHub
```
Comment (iteles) [inline README.md]: So great 😊

File: README.md (hunk 4 above)
Comment (mckennapsean) [reply README.md]: Thanks!

File: README.md (hunk 1 above)
Comment (mckennapsean) [reply README.md]: Cool, I can definitely take a quick editing pass on this for sure. Not sure how to best have line breaks on a bulleted list item in Markdown offhand though. I can fix this up and assign when ready!

**Comment by mckennapsean** [comment]
Cleaned up that one paragraph significantly, thanks @iteles !

I was unsure how to add newlines / breaks without manual HTML code to a Markdown bullet-list block. Feel free to add if you know how. But I think these improvements are ready to be merged.

**Pull request merged**